docs/next/api/plugins.md

```diff
@@ -8,38 +8,34 @@ canonicalUrl: /api/plugins
-|[HeaderTooltips](https://dev.handsontable.com/docs/next/api/header-tooltips/) |Deprecated feature. Display the header label in a tooltip.|
-|[HiddenColumns](https://dev.handsontable.com/docs/next/api/hidden-columns/) |Hide specific columns.|
-|[HiddenRows](https://dev.handsontable.com/docs/next/api/hidden-rows/) |Hide specific rows.|
-|[ManualColumnFreeze](https://dev.handsontable.com/docs/next/api/manual-column-freeze/) |Pin the columns to the left to keep them visible while scrolling.|
-|[ManualColumnMove](https://dev.handsontable.com/docs/next/api/manual-column-move/) |Drag a column to change its order in the data grid.|
-|[ManualColumnResize](https://dev.handsontable.com/docs/next/api/manual-column-resize/) | Drag the sizing handle to change the width of a column.|
-|[ManualRowMove](https://dev.handsontable.com/docs/next/api/manual-row-move/) |Drag a row to change its order in the data grid.|
-|[ManualRowResize](https://dev.handsontable.com/docs/next/api/manual-row-resize/) |Drag the sizing handle to change the height of a row.|
-|[MergeCells](https://dev.handsontable.com/docs/next/api/merge-cells/) |Span the cells across multiple rows or columns.|
-|[MultiColumnSorting](https://dev.handsontable.com/docs/next/api/multi-column-sorting/)| Sort the rows by multiple columns.|
-|[MultipleSelectionHandles](https://dev.handsontable.com/docs/next/api/multiple-selection-handles/)| This plugin extends the "core" selection feature to allow users to select multiple, non-adjacent ranges of cells.|
-|[NestedHeaders](https://dev.handsontable.com/docs/next/api/nested-headers/) |Create a nested, hierarchical structure of column headers to put your data into groups.|
-|[NestedRows](https://dev.handsontable.com/docs/next/api/nested-rows/) |Enable editable views of data sources with nested (tree/parent-child) structures.|
-|[ObserveChanges](https://dev.handsontable.com/docs/next/api/observe-changes/)| Enable the ObserveChanges plugin, which switches the table into one-way data binding. Changes applied to the data source will automatically be reflected in the table.|
-|[PersistentState](https://dev.handsontable.com/docs/next/api/persistent-state/)| Save the state of column sorting, column positions, and column sizes in local storage to preserve the table state between page reloads.|
-|[Search](https://dev.handsontable.com/docs/next/api/search/) |Search for specific values in the displayed data across the table.|
-|[TouchScroll](https://dev.handsontable.com/docs/next/api/touch-scroll/) |Scroll the table on mobile devices.|
-|[TrimRows](https://dev.handsontable.com/docs/next/api/trim-rows/) |Exclude the specific rows from being rendered and calculated by the data grid. This feature is mainly used internally, for example, in the filter plugin.|
-|[UndoRedo](https://dev.handsontable.com/docs/next/api/undo-redo/)|Undo or redo changes made by the user.|
+|[AutoColumnSize](@/api/AutoColumnSize.md) |Automatically adjusts the column's width to the size of the content.|
```

Comment:
@Siemienik were these changes not overwritten later? They are not on the current deployment demo.